onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -43,6 +42,7 @@ def __init__(self, model, per_channel, reduce_range, mode, static, weight_qType,
         self.extra_options = extra_options if extra_options is not None else {}
         self.q_matmul_const_b_only = 'MatMulConstBOnly' in self.extra_options and self.extra_options['MatMulConstBOnly']
         self.is_weight_symmetric = 'WeightSymmetric' not in self.extra_options or self.extra_options['WeightSymmetric']
+        self.is_activation_symmetric = 'ActivationSymmetric' not in self.extra_options or self.extra_options['WeightSymmetric']
```

Comment:
>WeightSymmetric [](http://example.com/codeflow?start=110&length=15)

It should be ActivationSymmetric

Reply:
Activation is different with weight. Weight can be trained to be symmetric, e.g., re-training with QAT, but activation can not. I'm thinking it'd better to make activation asymmetric by default.

Reply:
Thanks, fixed!